.github/workflows/a.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+echo "Running" | grep -c "Running"
```

Comment:
Is this script used, or did you intend to revert it?

Reply:
Added for testing, not needed.